src/main/java/org/primeframework/mvc/action/config/DefaultActionConfigurationBuilder.java

```diff
@@ -557,4 +561,62 @@ private Map<Class<?>, Object> getAdditionalConfiguration(Class<?> actionClass) {
     }
     return additionalConfiguration;
   }
+
+  private void validateNamedParameterAnnotations(Class<?> actionClass) {
+
+    var annotatedMethods = ReflectionUtils.findAllMethodsWithAnnotation(actionClass, NamedParameter.class);
+    var annotatedFields = ReflectionUtils.findAllFieldsWithAnnotation(actionClass, NamedParameter.class);
+    var allProperties = ReflectionUtils.findPropertyInfo(actionClass);
+    var allFields = ReflectionUtils.findFields(actionClass).keySet();
+    Map<String, Object> namedParameters = new HashMap<>();
+    annotatedMethods.forEach(method -> {
+      var name = method.getAnnotation(NamedParameter.class).name();
+      // find this method in the properties list, it should be there and be a valid property
+      var foundPropPair = allProperties.values()
+                                       .stream()
+                                       .filter(property -> !property.getName().equals(name))
+                                       .map(property -> Map.entry(property, property.getMethods()
+                                                                                    .values()
+                                                                                    .stream()
+                                                                                    .filter(pm -> pm.equals(method))
+                                                                                    .findFirst()))
+                                       .filter(pair -> pair.getValue().isPresent())
+                                       .findFirst()
+                                       .map(entry -> Map.entry(entry.getKey(), entry.getValue().get()))
+                                       .orElseThrow(() -> new PrimeException("The action class [" + actionClass.getSimpleName() + "] has a " +
```

Comment:
All other uses in this class use `actionClass` instead of `actionClass.getSimpleName()`. 

Also, to be more consistent with other messages, can we re-word this a bit? Same goes for the other messages. 

> The action class [FooAction] has a method annotated with... "


```
"The action class [" + actionClass + "] has a method annotated with "
+ NamedParameter.class.getSimpleName() + " for name  [" + method.getName() + "] .... "
```